scipy/stats/stats.py

```diff
@@ -2882,6 +2901,13 @@ def median_abs_deviation(x, axis=0, center=np.median, scale=1.0,
     will calculate the MAD around the mean - it will not calculate the *mean*
     absolute deviation.
 
+    The default `center` function is `numpy.median`, but it is wrapped in
+    code that will prevent it from generating a warning if it is given an
+    empty input.
+
+    The input array may contain `inf`, but if `center` returns `inf`, the
+    corresponding MAD for that data will be `nan`.
```

Comment:
good call out, do we already have (or do we need) a test for this case? 
could always be in a follow up PR with the `callable` check that I'm (now) thinking to write.

Reply:
I don't think we need a test; this is just a consequence of `inf - inf` producing `nan`.

Reply:
I was just thinking to have a test to confirm the behavior over future changes. Not critical, marking resolved w/o then.